cmd/skopeo/sync.go

```diff
@@ -166,36 +176,32 @@ func parseRepositoryReference(input string) (reference.Named, error) {
 // destinationReference creates an image reference using the provided transport.
```

Comment:
Something like
```suggestion
// destinationReference creates an image reference, combining the provided transport, the user-specified destination, and destSuffix containing a part of the source image’s name that should be used in the destination name.
```
otherwise the purpose of `destSuffix` is not obvious.

Possibly the order of arguments should then be changed to `transport, destination, destSuffix` so that it matches the usual order of the output.